pandas/tests/groupby/test_grouping.py

```diff
@@ -552,6 +552,17 @@ def test_list_grouper_with_nat(self):
         expected = {pd.Timestamp('2011-01-01'): 365}
         tm.assert_dict_equal(result.groups, expected)
 
+    @pytest.mark.parametrize('func', ['transform'])
```

Comment:
so you will need to handle the other operations, simply use a multi-parameterization, e.g.

```
@pytest.mark.parameterize(
    'func,expected',
    [
        ('transform', pd.Series(name=2, index=....)),
        ('agg', pd.DataFrame(.....)),
        ('apply', .....)
    ])
def test.....(self, func, expected):

```